src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -130,13 +139,24 @@ private function handleException(GetResponseForExceptionEvent $event)
                 break;
 
             default:
-                $this->logException($exception);
-                $this->renderTemplate('error', 500, $event);
-                break;
+                $this->renderErrorScreenByException($event);
```

Comment:
These changes are not really part of this PR, I'm unable to tell what the changes will cause. I think it will try to load language strings which is not intended here, but I'm not sure. Maybe @xtra needs to have a look.